### ISSUE TITLE
ELEKTRO-L TLM

### ISSUE THREAD

**Issue by sq7r0** [altillimity/SatDump#180]
Hi,

another issue found on decoding the Elektro L3 telemetry. Whole app freezes just after start the process. No error seen in logs
![2022-11-07 22_30_11-](https://user-images.githubusercontent.com/115694905/200420180-fed28870-0045-4ca9-8661-5f105cf4b2b6.png)
Saving the baseband and pass offline processing creates cadu file only. Is that correct?

cheers
Krzysztof

**Comment by Aang23** [comment]
Known issue. Do you have a baseband I could use to fix it up? Forgot where I put mines :-)

**Comment by sq7r0** [comment]
@Aang23 you are fast :D havent finished to upload the file and got reply xd Here is the baseband : https://cloud.egmont.pl/index.php/s/xjptIsYUtg0YUZe

**Comment by Aang23** [comment]
@sq7r0 check the latest commit once CI is done, should be fixed.

**Comment by sq7r0** [comment]
@Aang23 seems its fine now, works stable :) Do you plan any interpreter of this data?
![image](https://user-images.githubusercontent.com/115694905/200430008-02af029e-abef-4c73-ad1e-fbc7be7af675.png)

**Comment by Aang23** [comment]
Yes, I do! I have mostly reverse-engineered it during a solar-flare last time to somewhat reliably identify things. Though, do keep in mind all you will get out of it is radiation monitoring. Still useful and interesting, but not imagery!

**Comment by sq7r0** [comment]
@Aang23 yes thats perfect, it will be nice to combine with images taken with solar filter during flare eruptions, like this one i got some time ago ;)
![image](https://user-images.githubusercontent.com/115694905/200436315-31abc0d8-cf9a-4390-a8ea-306debf0a171.png)

**Comment by Aang23** [comment]
Indeed! It's some pretty nice data still, especially during this sort of events...

Awesome image by the way!
